frontend/public/landingfolder/landingpage.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
let's try and get this built in react! this is how the HTML should end up rendering but let's build this out as a react component/page